src/ui-kit/components/Button/index.js

```diff
@@ -0,0 +1,3 @@
+import DefaultButton from './Button';
```

Comment:
We do it through the default export, it should look like this: export {default} from ....